pkg/kubewarden/chart/kubewarden/policy-server/index.vue

```diff
@@ -1,104 +1,119 @@
 <script>
+import { reactive } from 'vue';
 import { _CREATE } from '@shell/config/query-params';
 import { CONFIG_MAP, SERVICE_ACCOUNT } from '@shell/config/types';
 import { allHash } from '@shell/utils/promise';
-import ResourceFetch from '@shell/mixins/resource-fetch';
+import { SECCOMP_OPTIONS } from '../../../components/PolicyServer/SeccompProfile.vue';
 
 import Loading from '@shell/components/Loading';
 import Tab from '@shell/components/Tabbed/Tab';
 import Labels from '@shell/components/form/Labels';
-import { SECCOMP_OPTIONS } from '../../../components/PolicyServer/SeccompProfile.vue';
 
-import General from './General';
-import SecurityContexts from './SecurityContexts';
-import Registry from './Registry/Index';
-import Verification from './Verification';
+import General from './General.vue';
+import SecurityContexts from './SecurityContexts.vue';
+import Registry from './Registry/Index.vue';
+import Verification from './Verification.vue';
+
+import ResourceFetch from '@shell/mixins/resource-fetch';
 
 export default {
   props: {
     mode: {
-      type:    String,
-      default: _CREATE
+      type: String,
+      default: _CREATE,
     },
 
     value: {
-      type:     Object,
-      default:  () => {}
-    }
+      type: Object,
+      default: () => ({}),
+    },
   },
 
   components: {
-    General, SecurityContexts, Labels, Loading, Tab, Registry, Verification
+    General,
+    SecurityContexts,
+    Labels,
+    Loading,
+    Tab,
+    Registry,
+    Verification,
   },
 
   mixins: [ResourceFetch],
 
   async fetch() {
     const hash = [];
 
-    if ( this.$store.getters['cluster/canList'](CONFIG_MAP) ) {
+    if (this.$store.getters['cluster/canList'](CONFIG_MAP)) {
       hash.push(this.$fetchType(CONFIG_MAP));
     }
 
-    if ( this.$store.getters['cluster/canList'](SERVICE_ACCOUNT) ) {
+    if (this.$store.getters['cluster/canList'](SERVICE_ACCOUNT)) {
       hash.push(this.$fetchType(SERVICE_ACCOUNT));
     }
 
     await allHash(hash);
   },
 
   data() {
-    if (!this.value.spec?.securityContexts) {
-      this.value.spec.securityContexts = {};
+    const chartValues = reactive({ ...this.value });
```

Comment:
Always interesting to see `reactive` in use. There were a few reactivity losses when we migrated to Vue3... Sometimes it's difficult to adapt the current implemented logic to match the new "reactivity" pattern without refactoring. Fortunately it wasn't that bad here